src/packages/solid/components/Row/Row.tsx

```diff
@@ -10,13 +10,18 @@ export interface RowProps extends IntrinsicNodeProps {
 }
 
 const Row: Component<RowProps> = (props: RowProps) => {
+  const multiFunction = (callback: Function | any, originalFunc: Function) => {
```

Comment:
lol - DRY - put this in utils and import it - we're going to need this everywhere